myapp/spec/models/user_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validaton' do
+    it 'create user successfully' do
+      user = build(:user)
+      expect(user).to be_valid
+    end
+
+    context 'when username is empty' do
+      before do
+        @user = build(:user, username: '')
+      end
+    
+      it 'user cannot be created without username' do
```

Comment:
I think `it does not create user` is more natural in a sentence than `it user cannot be created` wdyt? 👀